schemas/json/theme.json

```diff
@@ -428,9 +428,31 @@
 										"px",
 										"em",
 										"rem",
-										"vh",
```

Comment:
Looks like `vh` may have accidentally been removed here.

Reply:
Nope! Just moved below, on line 436 along with the other *vh units 👍

Reply:
My bad. I see it now 👍